catalog/app/containers/Bucket/Queries/json-editor.css

```diff
@@ -0,0 +1,3 @@
+.jsoneditor.jsoneditor {
```

Comment:
btw this can be scoped to the actual component requiring these overrides just by adding smth like this to the relevant `makeStyles` call:
```
root: {
  '& .jsoneditor': {
    ...overrides
  }
}
```